src/showingpreviously/cinemas/savoy.py

```diff
@@ -0,0 +1,116 @@
+import json
+import re
+from datetime import datetime, timedelta
+from bs4 import BeautifulSoup
+
+import showingpreviously.requests as requests
+from showingpreviously.model import ChainArchiver, CinemaArchiverException, Chain, Cinema, Screen, Film, Showing
+from showingpreviously.consts import UK_TIMEZONE, UNKNOWN_FILM_YEAR, STANDARD_DAYS_AHEAD
+
+
+CINEMAS_URL = 'http://www.savoycinemas.co.uk/'
+CHAIN = Chain('Savoy Cinemas')
+EVENTS_INLINE_PATTERN = re.compile(r'var\s+eventsInline\s*=\s*(?P<events_inline>(?:.|\n|\r)*?]);')
+
+
+def get_response(url: str) -> requests.Response:
+    r = requests.get(url)
+    if r.status_code != 200:
+        raise CinemaArchiverException(f'Got status code {r.status_code} when fetching URL {url}')
+    return r
+
+
+def parse_film_name(name: str):
+    for removal in ['U', '(PG)', '(12A)', '(12)', '(15)', '(18)', '2D']:
```

Comment:
Please add `4K` to this list and add `4K` as an attribute. An example of a film that would need this filtering is `LOTR: THE FELLOWSHIP OF THE RING 4K (20TH ANNIVERSARY)`